src/JSON.jl

```diff
@@ -28,4 +28,11 @@ using .Serializations: Serialization, CommonSerialization,
 # for pretty-printed (non-compact) output, JSONText must be re-parsed:
 Writer.lower(json::JSONText) = parse(json.s)
 
+function _precompile_()
+    x =  "{\"type\":\"callback\",\"data\":{\"callback\":1,\"result\":true,\"error\":false}}"
```

Comment:
Does this need a "if precompiling" check? https://github.com/timholy/Revise.jl/blob/5925b993136b8ae4041afa9032ae8e8b8cb9dbc9/src/precompile.jl#L13